doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -116,6 +116,7 @@ Security
 * Added:
 
   * Support for AES-CCM AEAD using CRACEN for the :zephyr:board:`nrf54lm20dk`.
+  * Support for the NIST SP 800-108 conformant HMAC KDF algorithm in CRACEN.
```

Comment:
```suggestion
  * Support for the HMAC KDF algorithm using CRACEN.
    The algorithm implementation is conformant to the NIST SP 800-108 Rev. 1 recommendation.
```
Please confirm the version of the recommendation. The one you mentioned originally is [not valid since 2022](https://csrc.nist.gov/pubs/sp/800/108/upd1/final).